submissions/Temu4/html_css_popup/index.html

```diff
@@ -0,0 +1,301 @@
+                  <div class="app-item__icon">
+                    <img src="img/apps-market.png" alt="Market" />
+                  </div>
+                  <div class="app-item__label"><p>Market</p></div>
+                </a>
+              </div>
+              <div class="app-item">
+                <a class="app-item__link" href="#">
+                  <div class="app-item__icon">
+                    <img src="img/apps-mail.png" alt="Bmail" />
+                  </div>
+                  <div class="app-item__label"><p>Bmail</p></div>
+                </a>
+              </div>
+              <div class="app-item">
+                <a class="app-item__link" href="#">
+                  <div class="app-item__icon">
+                    <img src="img/apps-disk.png" alt="Disk" />
+                  </div>
+                  <div class="app-item__label"><p>Disk</p></div>
+                </a>
+              </div>
+              <div class="app-item">
+                <a class="app-item__link" href="#">
+                  <div class="app-item__icon">
+                    <img src="img/apps-dairy.png" alt="Calendar" />
+                  </div>
+                  <div class="app-item__label"><p>Calendar</p></div>
+                </a>
+              </div>
+              <div class="app-item">
+                <a class="app-item__link" href="#">
+                  <div class="app-item__icon">
+                    <img src="img/apps-bussines.png" alt="Business" />
+                  </div>
+                  <div class="app-item__label"><p>Business</p></div>
+                </a>
+              </div>
+              <input
+                class="toggle-checkbox-more-list"
+                type="checkbox"
+                id="apps-more-list"
+              />
+              <!-- Button More -->
+              <div
+                class="app-box__more-button more-button"
+                tabindex="0"
+                target="more-button"
+              >
+                <label
+                  class="more-button__label"
+                  for="apps-more-list"
+                  tabindex=""
+                  >More
+                </label>
+              </div>
+
+              <!-- More apps list -->
+              <div class="apps-box__more-list">
+                <div class="divider"></div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-blogger.png" alt="Blogger" />
+                    </div>
+                    <div class="app-item__label"><p>Blogger</p></div>
+                  </a>
+                </div>
+                <div class="app-item">
+                  <a class="app-item__link" href="#">
+                    <div class="app-item__icon">
+                      <img src="img/apps-photos.png" alt="Photos" />
+                    </div>
```

Comment:
I'd rather use `background` or `background-image` to display icon and don't use contraction `<a><div><img>`, tag div is superfluous.

Reply:
`<div>` tags allow me to manipulate with elements in `flex` container.